src/routes/Home/components/LandingPage/Header/Header.jsx

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import { Icon } from 'react-fa';
+import { Link } from 'react-router';
```

Comment:
Is Link being used anywhere in the header? You can probably remove it from this line.